### DISCUSSION TITLE
How to re-activate an organization?

### DISCUSSION THREAD

**Discussion by jguelat** [opengisch/QField#3985] [Q&A]
Hello,

Our organizational account (Swiss_Ornithological_Institute) is set as inactive since the end of the beta period. I wanted to re-activate it and entered all the billing information but the account is still set as inactive. How can I re-activate it? I tried to change the subscription but I can only change the total storage.

Thanks for your help!

**Comment by suricactus** [comment]
Sorry for the invovenience. Working on a solution for you.

**Reply by jguelat** [reply]
OK, thanks!

**Reply by jguelat** [reply]
Hello,

Any news on this problem? Thanks in advance for your reply :-)

**Reply by lucienicolier** [reply]
Hi @jguelat,

Did you retry to subscribe by the billing setup in your organization?

**Reply by jguelat** [reply]
Hi @lucienicolier,

Thanks for your answer... I now see a new button "Finish Checkout" next to the billing information. I guess that's what I should use now...

**Reply by lucienicolier** [reply]
Exactly, when you have already begun to fill up the informations, it brings you where you stopped last time.

**Reply by jguelat** [reply]
Thanks!

**Reply by lucienicolier** [reply]
Glad to have helped :)
